apps/daimo-mobile/package.json

```diff
@@ -85,9 +85,9 @@
     "immutable": "^5.0.0-beta.5",
     "jest": "^29.5.0",
     "jest-expo": "~50.0.3",
-    "react": "18.2.0",
+    "react": "*",
     "react-error-boundary": "^4.0.12",
-    "react-native": "0.73.4",
+    "react-native": "*",
```

Comment:
why set these to "*"?

Reply:
version defined in the root `package.json`

there was some version hell involving the exact version of react-native required by Expo